compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -1076,6 +1076,21 @@ pub struct VarDebugInfo<'tcx> {
 // BasicBlock
 
 rustc_index::newtype_index! {
+    /// The unit of the MIR [control-flow graph][CFG].
+    ///
+    /// There is no branching (e.g., `if`s, function calls, etc.) within a basic block, which makes
+    /// it easier to do [data-flow analyses] and optimizations. Instead, branches are represented
+    /// as an edge in a graph between basic blocks.
+    ///
+    /// Basic blocks consist of a series of [statements][`Statement`], ending with a
+    /// [terminator][`Terminator`]. Basic blocks can have multiple predecessors and successors.
```

Comment:
You don't need backticks if it's not being shown to the reader.
```suggestion
    /// Basic blocks consist of a series of [statements][Statement], ending with a
    /// [terminator][Terminator]. Basic blocks can have multiple predecessors and successors.
```

Reply:
Good point, I'm so used to it :)